hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/BatchConsumer.java

```diff
@@ -151,6 +156,18 @@ public void updateTopic(Topic newTopic) {
         }
     }
 
+    @Override
+    public void commit() {
+        if (committer != null) {
```

Comment:
Committer is initialised outside of constructor. I should probably check in SerialConsumer too or write some precondition.